docs/lcdproc-user/drivers/futaba.docbook

```diff
@@ -0,0 +1,140 @@
+    any new drivers to work with lirc. It also work with
+    <application>scripts.xbmc.lcdproc</application> on
+    <application>Kodi</application>.</para>
+
+    <para>It comes pre-installed on Elonex Artisan, Fujitsu Scaleo E and FIC
+    Spectra Media Centre PCs <ulink
+    url="http://www.fic.com.tw/spectraviiv/drivers/SPECTRA%20ID1%20Viiv%20User%20Manual.pdf">FIC
+    Spectra Manual</ulink>.</para>
+
+    <note>
+      <itemizedlist>
+        <listitem>
+          <para>If the display crashes on start-up, this diff will solve the
+          problem: <ulink
+          url="https://github.com/ajw107/LibreELEC.tv/blob/master/packages/linux/patches/default/linux-061-add-elonex-artisan-vfd-support.patch">Linux
+          Diff</ulink>. I have submitted this patch for inclusion into the
+          standard Linux Kernel.</para>
+        </listitem>
+
+        <listitem>
+          <para>The patched version of
+          <application>scripts.xbmc.lcproc</application> can be found <ulink
+          url="https://github.com/ajw107/script.xbmc.lcdproc">here</ulink>,
+          this should be added upstream soon too.</para>
+        </listitem>
+      </itemizedlist>
+    </note>
+  </sect2>
+
+  <sect2 id="futaba-requirements">
+    <title>Requirements</title>
+
+    <para>The driver is based on the <ulink
+    url="http://libusb.info/"><filename>libusb</filename></ulink> USB library
+    (Versions 1.0 and 0.1), which should make it work with Linux, the
+    different BSB variants as well as Darwin/MacOS X.</para>
+
+    <note>
+      <para>When using a <filename>libusb</filename> based driver like
+      <code>futaba</code>, <application>LCDd</application> needs to be started
+      as root.</para>
+    </note>
+
+    <para>On Linux, the only kernel module required is the USB host controller
+    driver (<filename>uhci_hcd</filename> on the M300) to fire up the USB bus
+    to which the LCD is attached. For other operating systems, analogous
+    requirements apply.</para>
+
+    <para>Lastly, for <filename>libusb</filename> to work correctly, the
+    <filename>usbfs</filename> file system must be mounted on
+    <filename>/proc/bus/usb</filename>, e.g. using the command <code>mount -t
+    usbfs usbfs /proc/bus/usb</code> or by your system's default
+    configuration.</para>
+  </sect2>
+
+  <sect2 id="futaba-config">
+    <title>Configuration in LCDd.conf</title>
+
+    <sect3 id="futaba-config-section">
+      <title>[futaba]</title>
+
+      <variablelist>
+
+        <varlistentry>
+          <term><property>Size</property> =
+          <parameter>WIDTHxHEIGHT</parameter></term>
+
+          <listitem>
+            <para>Should not be changed from the default of 7x1.</para>
+          </listitem>
+        </varlistentry>
+
+        <varlistentry>
+          <term><property>TitleSpeed</property> =
+          <parameter><replaceable>TITLESPEED</replaceable></parameter></term>
+
```

Comment:
This is a parameter of the server, not the driver and need not be documented here. Also drop it from LCDd.conf together with the other server parameters. I missed that in the previous reviews.